src/lv_demo_music/lv_demo_music.h

```diff
@@ -15,7 +15,7 @@ extern "C" {
  *********************/
 #include "../../lv_examples.h"
 
-#if LV_USE_DEMO_MUSIC
+#if defined(LV_USE_DEMO_MUSIC)
```

Comment:
It won't disable the music player if `LV_USE_DEMO_MUSIC  0` in `lv_ex_conf.h`.

Reply:
That's a fair observation. I shall then submit issues on the  lv_sim_eclipse_sdl repo.